audit/projects/kubernetes-public/iam.json

```diff
@@ -93,6 +100,12 @@
       ],
       "role": "roles/logging.logWriter"
     },
+    {
+      "members": [
+        "group:gke-security-groups@kubernetes.io"
+      ],
+      "role": "roles/logging.privateLogViewer"
+    },
```

Comment:
Does this the PII log field / IP addresses on k8s.gcr.io?
I'd like to understand that role a bit better.